webrender/src/frame.rs

```diff
@@ -494,6 +502,15 @@ impl Frame {
                                    iframe_scroll_layer_id,
                                    iframe_reference_frame_id);
 
+        context.builder.push_scroll_layer(iframe_reference_frame_id,
```

Comment:
I think this is one place where the current terminology doesn't quite work. Reference frames have a layer in the ScrollTree, even though they themselves don't scroll. This will allow primitives to be positioned relative to the reference frame though. In this case, I don't think it is absolutely necessary, but I think it's useful to avoid having consistency for the moment.